wps_jobs/tests/weather_models/test_ecmwf.py

```diff
@@ -141,25 +123,9 @@ def test_ecmwf_process(mock_herbie_instance):
     assert ecmwf.files_processed == 130
 
 
-def test_ecmwf_process_model_run_prediction_not_found(mock_herbie_instance, monkeypatch):
```

Comment:
We may want to keep this test and adjust it to check that `exception_count` is incremented.